README.md

```diff
@@ -1,4 +1,15 @@
-# intern2021
+# Journey
+Journey is a learning experience running on [.NET Interactive Notebooks](https://github.com/dotnet/interactive) where teachers can design a step-by-step path for students. Along the student's journey, teachers can create challenges, provide customized feedback, and define rules to continue.
```

Comment:
"step-by-step"
 Just throwing something crazy out there. "challenge-by-challenge"
Can convey roughly a similar meaning. This might also help a little at contextualizing the term "challenge".